src/main/java/com/microsoft/sqlserver/jdbc/SecureStringUtil.java

```diff
@@ -137,16 +141,16 @@ byte[] getEncryptedBytes(char[] chars) throws SQLServerException {
      * @throws SQLServerException

```

Comment:
Nitpick - getting a javadoc complaint here and on line 137(@ param bytes). Please add a description for the param and throws if you're able to.